browser/resources/settings/brave_overrides/page_visibility.ts

```diff
@@ -82,6 +82,7 @@ function getPageVisibility () {
     // custom properties
     braveSync: !loadTimeData.getBoolean('isSyncDisabled'),
     braveWallet: loadTimeData.getBoolean('isBraveWalletAllowed'),
+    braveWeb3: loadTimeData.getBoolean('isBraveWalletAllowed'),
```

Comment:
There is `braveWallet` with same value above

Reply:
Well it seems it is for `pageVisibility.braveWeb3`

Reply:
I think this one is OK because you can't use Web3 without wallet, right?

Reply:
or hmm. Yeah there doesn't seem to be an individual policy for braveWeb3 🤔 

I think something is missing here

Reply:
I don't think we'll ever have a separate policy for web3 because it requires functionality in the wallet.  I'm removing the braveWeb3. I had the condition originally different but simplified it later.